etc/profile-a-l/chafa.profile

```diff
@@ -19,27 +19,24 @@ include disable-write-mnt.inc
 
 include whitelist-run-common.inc
 include whitelist-runuser-common.inc
-# Users may want to view images in /usr/share
 #include whitelist-usr-share-common.inc
 include whitelist-var-common.inc
 
 apparmor
 caps.drop all
 machine-id
 net none
+no3d
 nodvd
 nogroups
 noinput
 nonewprivs
 noroot
 nosound
-notv
-nou2f
-novideo
```

Comment:
?

Reply:
Based on your comment above, I thought those were only needed if the program required x11 access.
Since, I set `x11 none`, I thought they were no longer needed is this not the case?

Reply:
Why should they depend on graphical output? This are input devices.